perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -196,7 +196,7 @@ sub munge_overlapping_asset_bodies {
     my ($self, $bodies) = @_;
 
     # in_area will be true if the point is within the administrative area of Brent
-    my $in_area = $self->{c}->stash->{all_areas} && scalar(%{$self->{c}->stash->{all_areas}}) == 1 && (values %{$self->{c}->stash->{all_areas}})[0]->{id} eq $self->council_area_id->[0];
+    my $in_area = grep ($self->council_area_id->[0] == $_, keys %{$self->{c}->stash->{all_areas}});
```

Comment:
Presumably this fix also needs to apply to the other place Dave changed in the parent commit, or something else?